tests/trainer/test_trainer.py

```diff
@@ -988,3 +988,15 @@ def setup(self, stage):
     trainer.test(ckpt_path=None)
     assert trainer.stage == 'test'
     assert trainer.get_model().stage == 'test'
+
+
+def test_trainer_ddp_spawn_none_checkpoint(tmpdir):
+    model = EvalModelTemplate()
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        max_epochs=1,
+        checkpoint_callback=None,
+        distributed_backend="ddp_spawn"
+    )
+    result = trainer.fit(model)
```

Comment:
can you add a check here, that before and after fitting there is no checkpointing callback in the trainer? (just to ensure it really is disabled)?